[wasm] WBT failing to install with `Could not find IncludedWorkloadManifests.txt` because the file was renamed

[Build](https://dev.azure.com/dnceng-public/public/_build/results?buildId=272457&view=logs&j=1fa93050-f528-55d3-a351-f8bf9ce5adbf&t=d9dc6d3d-3442-5b70-1296-5b62e29ea209):

```
/__w/1/s/eng/testing/workloads-testing.targets(191,5): error : Could not find IncludedWorkloadManifests.txt in /__w/1/s/artifacts/bin/dotnet-none/ [/__w/1/s/src/mono/wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
##[error]eng/testing/workloads-testing.targets(191,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Could not find IncludedWorkloadManifests.txt in /__w/1/s/artifacts/bin/dotnet-none/
```
<!-- Error message template  -->
### Known Issue Error Message
Fill the error message using [known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md#how-to-fill-out-a-known-issue-error-section).

```json
{
  "ErrorMessage": "Could not find IncludedWorkloadManifests.txt",
  "BuildRetry": false
}
```
<!--Known issue error report start -->
### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[272472](https://dev.azure.com/dnceng-public/public/_build/results?buildId=272472)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/272472/logs/291)|dotnet/runtime#86187|
|[272457](https://dev.azure.com/dnceng-public/public/_build/results?buildId=272457)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/272457/logs/845)|dotnet/runtime#86188|
|[272420](https://dev.azure.com/dnceng-public/public/_build/results?buildId=272420)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/272420/logs/1570)|dotnet/runtime#85801|
|[272414](https://dev.azure.com/dnceng-public/public/_build/results?buildId=272414)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/272414/logs/808)|dotnet/runtime#85643|
|[272410](https://dev.azure.com/dnceng-public/public/_build/results?buildId=272410)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/272410/logs/108)|dotnet/runtime#86184|
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|5|5|5|
<!--Known issue error report end -->

--- a/src/tasks/WorkloadBuildTasks/InstallWorkloadFromArtifacts.cs
+++ b/src/tasks/WorkloadBuildTasks/InstallWorkloadFromArtifacts.cs
@@ -65,7 +65,6 @@ namespace Microsoft.Workload.Build.Tasks
                     throw new LogAsErrorException($"Cannot find {nameof(LocalNuGetsPath)}={LocalNuGetsPath} . " +
                                                     "Set it to the Shipping packages directory in artifacts.");
 
-                ExecuteHackForRenamedManifest();
                 if (!InstallAllManifests())
                     return false;
 
@@ -175,29 +174,6 @@ namespace Microsoft.Workload.Build.Tasks
             UpdateAppRef(req.TargetPath, req.Version);
 
             return !Log.HasLoggedErrors;
-        }
-
-        private void ExecuteHackForRenamedManifest()
-        {
-            // HACK - Because the microsoft.net.workload.mono.toolchain is being renamed to microsoft.net.workload.mono.toolchain.current
-            // but the sdk doesn't have the change yet.
-            string? txtPath = Directory.EnumerateFiles(Path.Combine(SdkWithNoWorkloadInstalledPath, "sdk"), "IncludedWorkloadManifests.txt",
-                                            new EnumerationOptions { RecurseSubdirectories = true, MaxRecursionDepth = 2})
-                                .FirstOrDefault();
-            if (txtPath is null)
-                throw new LogAsErrorException($"Could not find IncludedWorkloadManifests.txt in {SdkWithNoWorkloadInstalledPath}");
-
-            string stampPath = Path.Combine(Path.GetDirectoryName(txtPath)!, ".stamp");
-            if (File.Exists(stampPath))
-                return;
-
-            var lines = File.ReadAllLines(txtPath)
-                            .Select(line => line == "microsoft.net.workload.mono.toolchain"
-                                                ? "microsoft.net.workload.mono.toolchain.current"
-                                                : line);
-            File.WriteAllLines(txtPath, lines);
-
-            File.WriteAllText(stampPath, "");
         }
 
         private bool InstallAllManifests()

